src/server/render.ts

```diff
@@ -261,10 +261,19 @@ export async function render<Data>(
 
   const renderResults: [Plugin, PluginRenderResult][] = [];
 
+  const pluginRenderOptions = {
+    request: opts.request,
```

Comment:
I'm wondering if it makes more sense to go with `req`. That's the term used in `express` and `hono` and might be familiar to users coming from other frameworks.